archunit/src/main/java/com/tngtech/archunit/library/Architectures.java

```diff
@@ -1052,7 +1054,7 @@ public ArchRule allowEmptyShould(boolean allowEmptyShould) {
         @Override
         public OnionArchitecture as(String newDescription) {
             return new OnionArchitecture(domainModelPredicate, domainServicePredicate,
-                    applicationPredicate, adapterPredicates, ignoredDependencies,
+                    applicationPredicate, adapterPredicates, optionalLayers, ignoredDependencies,
```

Comment:
Unrelated to the issue reported in #1185, we also forgot to copy `allClassesAreContainedInArchitectureCheck`.

Reply:
I just added a commit to fix this when we're already on it...